test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -2178,6 +2179,12 @@ public static <T> T safeGet(Future<T> future) {
         }
     }
 
+    public static <T> Exception safeAwaitFailure(SubscribableListener<T> listener) {
```

Comment:
Since we don't use the generic type param `T` anywhere we can just use a wildcard instead:

```suggestion
    public static Exception safeAwaitFailure(SubscribableListener<?> listener) {
```